client/src/components/scorecard_page/ScorecardSummaryPanel.vue

```diff
@@ -152,20 +145,18 @@ export default defineComponent({
 <style scoped lang='scss'>
 @import 'src/assets/styles/global';
 @import '@blueconduit/copper/scss/01_settings/design-tokens';
-@import 'bulma/sass/layout/section';
-@import 'bulma/sass/elements/container';
 
-.section {
-  background-color: $navy-blue;
+.section.has-text-centered {
+  // Use a semi-opaque gradient to tint the background image for better text readability.
+  background-image: linear-gradient(rgba(4, 86, 155, 0.75), rgba(4, 86, 155, 0.75)),
+  url('~@/assets/media/understand-score@2x.png'), url('~@/assets/media/understand-score.png');
```

Comment:
(same comment as https://github.com/BlueConduit/open-data-platform/pull/188#discussion_r978064031)

Reply:
Removed 2x and 3x - Didn't realize how big they were